### ISSUE TITLE
Is able to temporarily disable context change?

### ISSUE THREAD

**Issue by mcpeng** [lukas-reineke/indent-blankline.nvim#472]
Is it possible to temporarily disable context guide change while screen scrolling or cursor moving? I thinks that's useful for large context( higher than one screen ).

like VSCode does:
<img width="498" alt="vsc" src="https://user-images.githubusercontent.com/2771843/185201863-a60d7bed-ba30-4401-9a57-eff054855dec.png">

"Vim requires the cursor to be in the current screen at all times" 🤔
https://stackoverflow.com/a/3458731

**Comment by lukas-reineke** [comment]
I'm not sure I understand the question.
You want to lock the context so it doesn't change even when you move outside?

**Comment by mcpeng** [comment]
> I'm not sure I understand the question. You want to lock the context so it doesn't change even when you move outside?

Yes, when the context higher than the screen; with locking the context guide line, I can easily check where  the context starts or ends, or check whether some code lines are under the context.

Now, I use `colorcolumn` / `multi colored guide lines` for that situation, but it's not perfect.

**Comment by lukas-reineke** [comment]
I see. This is currently not supported. I'll think about adding it in the future.

**Comment by mcpeng** [comment]
😸  Thanks for your great job of **indent-blankline.nvim**
